xml/System/Array.xml

```diff
@@ -6350,7 +6350,7 @@ int[,,] TDArray = new int[1,1,1];
         <param name="array">The one-dimensional <see cref="T:System.Array" /> to reverse.</param>
         <param name="index">The starting index of the section to reverse.</param>
         <param name="length">The number of elements in the section to reverse.</param>
-        <summary>Reverses the sequence of the elements in a range of elements in the one-dimensional <see cref="T:System.Array" />.</summary>
+        <summary>Reverses the sequence of a subset of the elements in the one-dimensional <see cref="T:System.Array" />.</summary>
```

Comment:
```suggestion
        <summary>Reverses the sequence of a subset of the elements in the one-dimensional array.</summary>
```

Reply:
You reverted the original text.

Reply:
Nope, just changed the cross-reference for plain text.

Reply:
Nevermind. My bad.